openzwavemqtt/models/value.py

```diff
@@ -73,7 +73,7 @@ def index(self) -> ValueIndex:
             # Nothing breaks but it isn't very pretty
             return ValueIndex(self.data.get("Index"))
         except ValueError:
-            return ValueIndex.UNKNOWN
+            return int(self.data.get("Index"))
```

Comment:
I'd like this to have a consistent return value. So for now I'm thinking we should just always return the integer.

We can return the enum again later if we complete the enums somehow.

Reply:
I vote for dropping the Enum for now.